src/main/resources/application.properties

```diff
@@ -18,3 +18,37 @@
 #
 
 
+#
+# Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+##
+#Springdoc swagger configuration
+springdoc.api-docs.enabled=true
+springdoc.api-docs.path=/docs/api-docs
+springdoc.swagger-ui.disable-swagger-default-url=true
+springdoc.swagger-ui.path=/ui/swagger-ui
+springdoc.swagger-ui.show-common-extensions=true
+springdoc.swagger-ui.csrf.enabled=true
```

Comment:
CSRF doesn't need to be enabled if its not enabled in the security configuration of the API. If you plan to use OAUTH2 security scheme to secure the application then CSRF is not needed in any case.

Reply:
Done